tests/server.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test } from "@playwright/test";
+import {
+  ServerSettingType,
+  activateSetting,
+  resetSetting,
+  updateSetting,
+  visitServerSettings,
+} from "./helpers/server";
+
+const settings: {
+  settingName: string;
+  settingType: ServerSettingType;
+  content?: string;
+}[] = [
+  { settingName: "acme.agree_tos", settingType: "checkbox" },
```

Comment:
Shall we drop the `setting` prefix from the properties? Or maybe with 3 settings left and much shorter test function, the array is not needed and simpler to write it out directly in the test?

Reply:
I think let's keep the array, it doesn't hurt to keep this structure in case if we want to test more settings at a later stage then we can extend easily?